src/logic/FetchRemotes.ts

```diff
@@ -96,26 +109,26 @@ export async function fetchExtensionManifest(contents_url: string, branch: strin
         repo,
         branch: selectedBranch,
 
-        imageURL: manifest.preview && manifest.preview.startsWith("http")
-          ? manifest.preview
-          : `https://raw.githubusercontent.com/${user}/${repo}/${selectedBranch}/${manifest.preview}`,
+        imageURL:
+          manifest.preview && manifest.preview.startsWith("http")
+            ? manifest.preview
+            : `https://raw.githubusercontent.com/${user}/${repo}/${selectedBranch}/${manifest.preview}`,
```

Comment:
Same note as above for these

Reply:
You wish to split a string into 2 separate lines?

Reply:
Same answer as above. :/